k8s/apis/k8s/v1alpha1/vineyardd_types.go

```diff
@@ -0,0 +1,193 @@
+/** Copyright 2020-2022 Alibaba Group Holding Limited.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1alpha1 contains API Schema definitions for the k8s v1alpha1 API group
+package v1alpha1
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// MetricConfig holds all metrics configuration
+type MetricConfig struct {
+	// represent the metric's image
+	// +kubebuilder:validation:Optional
+	Image string `json:"image,omitempty"`
+	// the policy about pulling image
+	// +kubebuilder:validation:Optional
+	ImagePullPolicy string `json:"imagePullPolicy,omitempty"`
+}
```

Comment:
It is wired to call `Image` and `ImagePullPocliy` as "metric".

Reply:
The `Image` and `ImagePullPocliy ` is the field in the Metric Container, and it is used to differentiate the vineyardd's configuration.